tox.ini

```diff
@@ -38,7 +39,10 @@ commands =
   mypy: mypy --namespace-packages opentelemetry-api/src/opentelemetry/
 ; For test code, we don't want to enforce the full mypy strictness
   mypy: mypy --namespace-packages --config-file=mypy-relaxed.ini opentelemetry-api/tests/
-  test: python -m unittest discover
+  test: pytest --cov {envsitepackagesdir}/opentelemetry
```

Comment:
there's a few unit tests for opentelemetry-api as well. I feel like we should report coverage on those.

Reply:
Hello @toumorokoshi,

Coverage on API is being reported for every `*-test-*` due to settings in `pytest.ini`.
Three lines those follows this one are for additional reporting.

![opentelemetry-coverage](https://user-images.githubusercontent.com/5607572/64511392-e2c22580-d2ec-11e9-9c71-c325b6569bb8.png)

You can find the full build log for Python 3.7 here:
https://travis-ci.org/open-telemetry/opentelemetry-python/jobs/581936178

Also, please let me know if I'm missing something or there is any misunderstanding on my side.

Thanks!

Reply:
So the combined coverage report (e.g. considering that the ext-wsgi tests will also increase coverage of the API code) is available under https://codecov.io/github/open-telemetry/opentelemetry-python/commit/de8f21ecf59322ddbd71c145c74326824a16bd30? The API package's trace module seems to be missing in that report.

Reply:
> So the combined coverage report (e.g. considering that the ext-wsgi tests will also increase coverage of the API code) is available under codecov.io/github/open-telemetry/opentelemetry-python/commit/de8f21ecf59322ddbd71c145c74326824a16bd30?

Yes, it is.

> The API package's trace module seems to be missing in that report.

I've tried to investigate and fix this, but it seems like this is a bug on the Codecov side.
I'll report it tomorrow.

Reply:
I've described this bug [here](https://codecov.slack.com/archives/C04KHQ1C8/p1568160835019200).

Reply:
@Jamim can you post the text for people like me that can't log into that slack?

Reply:
The issue is already resolved in the current version of changes, but here is that text:

> Hi there,
I found that some files are not visible on [Codecov.io](http://codecov.io/) for some reason.
I tried to investigate this but I didn't find a reason why some files are shown and other files are not.
>
> PR: https://github.com/open-telemetry/opentelemetry-python/pull/128
Build: https://travis-ci.org/open-telemetry/opentelemetry-python/builds/581936174
Codecov: https://codecov.io/gh/open-telemetry/opentelemetry-python/tree/de8f21ecf59322ddbd71c145c74326824a16bd30On https://codecov.io/gh/open-telemetry/opentelemetry-python/tree/de8f21ecf59322ddbd71c145c74326824a16bd30/opentelemetry-api/src/opentelemetry page I'm expecting to see
```
context/
distributedcontext/
logs/
metrics/
resources/
trace/
loader.py
types.py
version.py
```
> but I can see only
```
context/
distributedcontext/
loader.py
types.py
```
> And corresponding reports contain stats for missing files, e.g.
https://codecov.io/codecov/v4/raw/2019-09-07/E6CBE819EACB252EDCB65A9B1EE32DAB/de8f21ecf59322ddbd71c145c74326824a16bd30/675278e2-de86-49ca-b141-f260fb05b38c.txt contains stats for API's `metrics/__init__.py`.